### PR TITLE
18805 Changed amalgamation to amalgamationApplication

### PR DESCRIPTION
**Pull request by JazzarKarim**
*Issue #:* /bcgov/entity#18805

*Description of changes:*
- Need this change now as I'm using the Allowable Actions (PR has been merged: https://github.com/bcgov/lear/commit/9ef825e0e850f6ae713093cbcb7509a5767ec072)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/enums/filing-types.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ export enum FilingTypes {
   ALTERATION = 'alteration',
   AGM_EXTENSION = 'agmExtension',
   AGM_LOCATION_CHANGE = 'agmLocationChange',
-  AMALGAMATION = 'amalgamation',
+  AMALGAMATION = 'amalgamationApplication',
```
Comment (severinbeauvais) [inline src/enums/filing-types.ts:6]: Do you think the key should be renamed as well?

Comment (severinbeauvais) [reply src/enums/filing-types.ts:6]: (The UIs will need to import the updated shared enum anyway.)

Comment (JazzarKarim) [reply src/enums/filing-types.ts:6]: Was thinking about that. Yes, I think that would be a good idea. On it!

**Review by severinbeauvais — APPROVED** [review]

**Comment by severinbeauvais** [comment]
```
lerna notice
Successfully published:
 - @bcrs-shared-components/action-chip@1.1.38
 - @bcrs-shared-components/approval-type@1.0.44
 - @bcrs-shared-components/base-address@2.0.14
 - @bcrs-shared-components/breadcrumb@2.1.38
 - @bcrs-shared-components/business-lookup@1.2.7
 - @bcrs-shared-components/certify@2.1.38
 - @bcrs-shared-components/completing-party@2.1.41
 - @bcrs-shared-components/contact-info@1.2.38
 - @bcrs-shared-components/correct-name@1.0.32
 - @bcrs-shared-components/court-order-poa@3.0.34
 - @bcrs-shared-components/date-picker@1.2.41
 - @bcrs-shared-components/effective-date-time@1.1.41
 - @bcrs-shared-components/enums@1.1.4
 - @bcrs-shared-components/fee-summary@2.0.12
 - @bcrs-shared-components/folio-number-input@1.1.38
 - @bcrs-shared-components/interfaces@1.1.4
 - @bcrs-shared-components/jurisdiction@1.0.2
 - @bcrs-shared-components/limited-restoration-panel@2.0.22
 - @bcrs-shared-components/mixins@1.1.32
 - @bcrs-shared-components/nature-of-business@1.2.37
 - @bcrs-shared-components/relationships-panel@1.0.32
 - @bcrs-shared-components/services@1.0.13
 - @bcrs-shared-components/share-structure@2.2.15
 - @bcrs-shared-components/staff-comments@1.3.39
 - @bcrs-shared-components/staff-payment@2.1.38
lerna success published 25 packages
```

**Comment by JazzarKarim** [comment]
> ```
> lerna notice
> Successfully published:
>  - @bcrs-shared-components/action-chip@1.1.38
>  - @bcrs-shared-components/approval-type@1.0.44
>  - @bcrs-shared-components/base-address@2.0.14
>  - @bcrs-shared-components/breadcrumb@2.1.38
>  - @bcrs-shared-components/business-lookup@1.2.7
>  - @bcrs-shared-components/certify@2.1.38
>  - @bcrs-shared-components/completing-party@2.1.41
>  - @bcrs-shared-components/contact-info@1.2.38
>  - @bcrs-shared-components/correct-name@1.0.32
>  - @bcrs-shared-components/court-order-poa@3.0.34
>  - @bcrs-shared-components/date-picker@1.2.41
>  - @bcrs-shared-components/effective-date-time@1.1.41
>  - @bcrs-shared-components/enums@1.1.4
>  - @bcrs-shared-components/fee-summary@2.0.12
>  - @bcrs-shared-components/folio-number-input@1.1.38
>  - @bcrs-shared-components/interfaces@1.1.4
>  - @bcrs-shared-components/jurisdiction@1.0.2
>  - @bcrs-shared-components/limited-restoration-panel@2.0.22
>  - @bcrs-shared-components/mixins@1.1.32
>  - @bcrs-shared-components/nature-of-business@1.2.37
>  - @bcrs-shared-components/relationships-panel@1.0.32
>  - @bcrs-shared-components/services@1.0.13
>  - @bcrs-shared-components/share-structure@2.2.15
>  - @bcrs-shared-components/staff-comments@1.3.39
>  - @bcrs-shared-components/staff-payment@2.1.38
> lerna success published 25 packages
> ```

Thank you Sev!

**Pull request merged**